### PR TITLE
docs: descriptions about externalTarget in NetworkChaos

### PR DESCRIPTION
**Pull request by STRRL**
Signed-off-by: STRRL <str_ruiling@outlook.com>

Appending description about field `externalTargets` in Network Chaos

### CODE REVIEW

**Comment by STRRL** [comment]
PTAL
/cc @YangKeao @Hexilee

File: docs/simulate-network-chaos-on-kubernetes.md [hunk 1]
```diff
@@ -147,6 +147,7 @@ Before creating NetworkChaos experiments, ensure the following:
 | value | string | Provides a parameter for the `mode` configuration, depending on `mode`. For example, when `mode` is set to `fixed-percent`, `value` specifies the percentage of Pods. | None | No | 1 |
 | containerNames | []string | Specifies the name of the container into which the fault is injected. | None | No | ["nginx"] |
 | selector | struct | Specifies the target Pod. For details, refer to [Define the experiment scope](./define-chaos-experiment-scope.md). | None | Yes |  |
+| externalTargets | []string | Represents network targets outside k8s, IPv4 addresses or domains. Only works with `direction: to`. | None | No | 1.1.1.1, www.google.com |
```
Comment (en-jin19) [inline docs/simulate-network-chaos-on-kubernetes.md:150]:
```suggestion
| externalTargets | []string | Indicates the network targets except for Kubernetes, which can be IPv4 addresses or domains. This parameter only works with `direction: to`. | None | No | 1.1.1.1, www.google.com |
```

File: i18n/zh/docusaurus-plugin-content-docs/current/simulate-network-chaos-on-kubernetes.md [hunk 2]
```diff
@@ -145,6 +145,7 @@ NetworkChaos 用于模拟集群中网络故障的场景，目前支持以下几
 | value | string | 取决与 `mode` 的配置，为 `mode` 提供对应的参数。例如，当你将 `mode` 配置为 `fixed-percent` 时，`value` 用于指定 Pod 的百分比 | 无 | 否 | 1 |
 | containerNames | []string | 指定注入的容器名称 | 无 | 否 | ["nginx"] |
 | selector | struct | 指定注入故障的目标 Pod，详情请参考[定义实验范围](./define-chaos-experiment-scope.md) | 无 | 是 |  |
+| externalTargets | []string | 表示 k8s 之外的网络目标, 可以是 IPv4 地址或者域名。只能与 `direction: to` 一起工作。 | 无 | 否 | 1.1.1.1, www.google.com |
```
Comment (en-jin19) [inline i18n/zh/docusaurus-plugin-content-docs/current/simulate-network-chaos-on-kubernetes.md:148]:
```suggestion
| externalTargets | []string | 表示 Kubernetes 之外的网络目标, 可以是 IPv4 地址或者域名。只能与 `direction: to` 一起工作。 | 无 | 否 | 1.1.1.1, www.google.com |
```

File: i18n/zh/docusaurus-plugin-content-docs/version-2.0.5/simulate-network-chaos-on-kubernetes.md [hunk 3]
```diff
@@ -145,6 +145,7 @@ NetworkChaos 用于模拟集群中网络故障的场景，目前支持以下几
 | value | string | 取决与 `mode` 的配置，为 `mode` 提供对应的参数。例如，当你将 `mode` 配置为 `fixed-percent` 时，`value` 用于指定 Pod 的百分比 | 无 | 否 | 1 |
 | containerNames | []string | 指定注入的容器名称 | 无 | 否 | ["nginx"] |
 | selector | struct | 指定注入故障的目标 Pod，详情请参考[定义实验范围](./define-chaos-experiment-scope.md) | 无 | 是 |  |
+| externalTargets | []string | 表示 k8s 之外的网络目标, 可以是 IPv4 地址或者域名。只能与 `direction: to` 一起工作。 | 无 | 否 | 1.1.1.1, www.google.com |
```
Comment (en-jin19) [inline i18n/zh/docusaurus-plugin-content-docs/version-2.0.5/simulate-network-chaos-on-kubernetes.md:148]:
```suggestion
| externalTargets | []string | 表示 Kubernetes 之外的网络目标, 可以是 IPv4 地址或者域名。只能与 `direction: to` 一起工作。 | 无 | 否 | 1.1.1.1, www.google.com |
```

File: versioned_docs/version-2.0.5/simulate-network-chaos-on-kubernetes.md [hunk 4]
```diff
@@ -147,6 +147,7 @@ Before creating NetworkChaos experiments, ensure the following:
 | value | string | Provides a parameter for the `mode` configuration, depending on `mode`. For example, when `mode` is set to `fixed-percent`, `value` specifies the percentage of Pods. | None | No | 1 |
 | containerNames | []string | Specifies the name of the container into which the fault is injected. | None | No | ["nginx"] |
 | selector | struct | Specifies the target Pod. For details, refer to [Define the experiment scope](./define-chaos-experiment-scope.md). | None | Yes |  |
+| externalTargets | []string | Represents network targets outside k8s, IPv4 addresses or domains. Only works with `direction: to`. | None | No | 1.1.1.1, www.google.com |
```
Comment (en-jin19) [inline versioned_docs/version-2.0.5/simulate-network-chaos-on-kubernetes.md:150]:
```suggestion
| externalTargets | []string | Indicates the network targets except for Kubernetes, which can be IPv4 addresses or domains. This parameter only works with `direction: to`. | None | No | 1.1.1.1, www.google.com |
```

**Review by YangKeao — APPROVED** [review]
LGTM

**Review by Hexilee — APPROVED** [review]
LGTM

**Review by en-jin19 — APPROVED** [review]
LGTM

**Pull request merged**